swagger.yaml

```diff
@@ -573,6 +573,87 @@ paths:
           schema:
             $ref: '#/definitions/error'
 
+  /clusters/{cluster_id}/ignition-config:
+    get:
+      tags:
+        - installer
+      summary: Get the cluster ignition config
+      operationId: GetClusterIgnitionConfig
+      parameters:
+        - in: path
+          name: cluster_id
+          type: string
+          format: uuid
+          required: true
+      responses:
+        200:
+          description: Success.
+          schema:
+            type: string
+        401:
+          description: Unauthorized.
+          schema:
+            $ref: '#/definitions/error'
+        403:
+          description: Forbidden.
+          schema:
+            $ref: '#/definitions/error'
+        404:
+          description: Error.
+          schema:
+            $ref: '#/definitions/error'
+        405:
+          description: Method Not Allowed.
+          schema:
+            $ref: '#/definitions/error'
+        500:
+          description: Error.
+          schema:
+            $ref: '#/definitions/error'
+    patch:
+      tags:
+        - installer
+      summary: Override values in the initial ignition config
+      operationId: UpdateClusterIgnitionConfig
+      parameters:
+        - in: path
+          name: cluster_id
+          type: string
+          format: uuid
+          required: true
+        - in: body
+          name: ignition-config-params
+          required: true
+          schema:
+            type: string
+      responses:
+        201:
+          description: Success.
+        400:
+          description: Error.
+          schema:
+            $ref: '#/definitions/error'
+        401:
+          description: Unauthorized.
+          schema:
```

Comment:
401-403 should be infra_error https://github.com/openshift/assisted-service/blob/master/swagger.yaml#L2355

Reply:
Fixed existing endpoints in https://github.com/openshift/assisted-service/pull/335